js/PixelPainter.js

```diff
@@ -0,0 +1,125 @@
+window.PixelPainter = (function(){
+
+  let paint = {};
+
+  const doc = document;
```

Comment:
Since you are using an IIFE, passing `document ` in as an argument would be better form.

```
(function (doc) { // code })(document);
```